PyTado/interface/api/my_tado.py

```diff
@@ -525,6 +525,18 @@ def get_eiq_meter_readings(self):
         request.endpoint = Endpoint.EIQ
 
         return self._http.request(request)
+        
+    def get_eiq_consuption_overview(self):
+        """
+        Get Energy IQ consuption overview
+        """
+        
+        request = TadoRequest()
+        request.command = "consumptionOverview"
+        request.action = Action.GET
+        request.endpoint = Endpoint.EIQ
+        
+        return self._http.request(request)
```

Comment:
I would make all parameters to the call as parameters to the method:


```suggestion
    def get_eiq_consuption_overview(self, year, month, country):
        """
        Get consumption overview data for a specific month

        Args:
            year (str): The year to get the consumption overview for.
            month (str): The month to get the consumption overview for.
            country (str): Country code.

        Returns:
            dict: Consumption overview data for the specified month
        """

        request = TadoRequest()
        request.command = "consumptionOverview"
        request.action = Action.GET
        request.endpoint = Endpoint.EIQ
        request.params = {"month": f"{year}-{month}", "country": country}

        return self._http.request(request)
```

Reply:
we could also add the year-month as a single default parameter:

```suggestion
    def get_eiq_consuption_overview(self, date=datetime.datetime.now().strftime("%Y-%m"):
        """
        Get consumption overview data for a specific month

        Args:
            date (str): The year-month to get the consumption overview for.

        Returns:
            dict: Consumption overview data for the specified month
        """

        request = TadoRequest()
        request.command = "consumptionOverview"
        request.action = Action.GET
        request.endpoint = Endpoint.EIQ
        request.params = {"month": f"{date}"}

        return self._http.request(request)```